files/zh-tw/mozilla/add-ons/webextensions/api/storage/storagearea/get/index.md

```diff
@@ -11,9 +11,11 @@ translation_of: Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get
 
 ## 語法
 
-    let gettingItem = browser.storage.<storageType>.get(
-      keys    // null, string, object or array of strings
-    )
+```
```

Comment:
````suggestion
```js
````

Reply:
I had started to add this one, but it's really pseudo code rather than real JS

Reply:
Yes. And `js` is added in en-US. So I just follow that:

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/storage/StorageArea/get